packages/signer-p12/dist/P12Signer.d.ts

```diff
@@ -14,6 +14,11 @@ export class P12Signer extends Signer {
         asn1StrictParsing: boolean;
     };
     cert: any;
+    /**
+     * @param {Buffer} pdfBuffer
+     * @returns {Buffer}
+     */
+    sign(pdfBuffer: Buffer): Buffer;
```

Comment:
You shouldn't manually edit the .d.ts files. They are auto-generated at build using JSDoc. Instead you should edit the JSDoc in `src/P12Signer.js` and run `yarn build`.

Reply:
That's what I did 😬 

I think because the typings are narrowed by the implementation, this has been updated by the build system.